packages/custom-token-bridging/scripts/exec-viaDapp.js

```diff
@@ -1,14 +1,13 @@
-const { providers, Wallet } = require('ethers')
-const { ethers } = require('hardhat')
-const { Bridge, networks } = require('arb-ts')
+const { providers, Wallet } = require('ethers') 
 const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
+const { getL2Network } = require('arb-ts')
+const { L1ToL2MessageGasEstimator } = require('arb-ts/dist/lib/message/L1ToL2MessageGasEstimator')
```

Comment:
```suggestion
const { L1ToL2MessageGasEstimator } = require('arb-ts')
```

Reply:
It looks like we dont actually export this from index yet, I'll add it to arb-ts